packages/sdk/src/wallet/core/providers/WalletProvider.ts

```diff
@@ -29,35 +29,23 @@ export class WalletProvider<
   /**
    * Create a new smart wallet
    * @description Creates a smart wallet and attempts to deploy it across all supported chains.
-   * The wallet address is deterministically calculated from owners and nonce. The signer must
-   * be included in the owners array. Deployment failures on individual chains do not prevent
+   * The wallet address is deterministically calculated from signers and nonce. The signer must
+   * be included in the signers array. Deployment failures on individual chains do not prevent
    * wallet creation - they are reported in the result.
    * @param params - Smart wallet creation parameters
-   * @param params.owners - Array of owners for the smart wallet (addresses or WebAuthn public keys)
-   * @param params.signer - Local account used for signing transactions (must be in owners array)
+   * @param params.signers - Array of signers for the smart wallet
+   * @param params.signer - Local account used for signing transactions (must be in signers array)
    * @param params.nonce - Optional nonce for smart wallet address generation (defaults to 0)
    * @param params.deploymentChainIds - Optional chain IDs to deploy the wallet to.
    * If not provided, the wallet will be deployed to all supported chains.
    * @returns Promise resolving to deployment result containing:
    * - `wallet`: The created SmartWallet instance
    * - `deployments`: Array of deployment results with chainId, receipt, success flag, and error
-   * @throws Error if signer is not included in the owners array
+   * @throws Error if signer is not included in the signers array
    */
   async createSmartWallet(
     params: CreateSmartWalletOptions,
   ): Promise<SmartWalletCreationResult<SmartWallet>> {
-    const { owners, signer } = params
-
-    if (
-      owners.filter(
-        (owner) =>
-          typeof owner === 'string' &&
-          getAddress(owner) === getAddress(signer.address),
-      ).length === 0
-    ) {
-      throw new Error('Signer must be in the owners array')
-    }
-
```

Comment:
we already do this when calling `DefaultSmartWallet.create` so this is duplicating work and just introduces unnecessary surface area for bugs